stdlib/ballerina-websub/src/main/java/org/ballerinalang/net/websub/WebSubDispatcher.java

```diff
@@ -36,28 +36,28 @@
      * This method finds the matching resource for the incoming request.
      *
      * @param servicesRegistry  the WebSubServicesRegistry including registered WebSub Services
-     * @param httpCarbonMessage incoming message.
+     * @param inboundMessage incoming message.
      * @return matching resource.
      */
      static HttpResource findResource(WebSubServicesRegistry servicesRegistry,
-                                            HTTPCarbonMessage httpCarbonMessage) {
-        HttpResource resource = null;
-        String protocol = (String) httpCarbonMessage.getProperty(HttpConstants.PROTOCOL);
+                                            HTTPCarbonMessage inboundMessage) {
+        HttpResource resource;
+        String protocol = (String) inboundMessage.getProperty(HttpConstants.PROTOCOL);
         if (protocol == null) {
             throw new BallerinaConnectorException("protocol not defined in the incoming request");
         }
 
         try {
             HttpService service = HttpDispatcher.findService(servicesRegistry,
-                                                             httpCarbonMessage);
+                                                             inboundMessage);
```

Comment:
Can't we just write this in a single line? 
HttpService service = HttpDispatcher.findService(servicesRegistry, inboundMessage);

Reply:
Ack